code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1062,6 +1062,10 @@ GLOBAL_LIST_EMPTY(features_by_species)
 			return "ADJ"
 		if(BODY_FRONT_LAYER)
 			return "FRONT"
+		//SKYRAT EDIT ADDITION BEGIN
+		if(BODY_FRONT_UNDER_CLOTHES)
+			return "FRONT"
+		//SKYRAT EDIT ADDITION END
```

Comment:
:-1:
I'm not comfortable adding any additional rendering layers to humans at this time given the insane overtime we're dealing with every time a human gets getFlatIcon'd.

Reply:
We can’t do anything about that, taurs code is terrible and this is easiest possible way to fix it. Not the best, but best it reworking taur completely. We cannot do that.